src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -148,12 +156,8 @@ export class OutlineRegionPicker extends LitElement {
 
   _handleCreateServerTap(): void {
     this.isServerBeingCreated = true;
```

Comment:
That's application logic. Let's make the parent set this instead.

Reply:
Done